src/queueloader.cpp

```diff
@@ -27,7 +27,7 @@ void queueloader::reload(std::vector<download>& downloads, bool remove_unplayed)
 	std::vector<download> dltemp;
 	std::fstream f;
 
-	for (auto dl : downloads) {
+	for (auto& dl : downloads) {
```

Comment:
With methods of `download` constified, it seems `auto&` in this file could be changed to `const auto&`.